capsules/extra/src/ethernet_app_tap.rs

```diff
@@ -213,7 +213,7 @@ pub const DRIVER_NUM: usize = capsules_core::driver::NUM::EthernetTAP as usize;
 
 /// Maximum size of a packet which can be transmitted over the underlying
 /// [`EthernetAdapter`] device. Currently hard-coded to `1522` bytes.
-pub const MAX_MTU: usize = 1522;
+pub const MAX_MTU: usize = 1524;
```

Comment:
Receive descriptors allow buffer lengts multiple of 4, 8 or 16, depending how the DMA is configured.

Reply:
This is a good point. We should reconsider this in the design when we revisit the packet buffer passing story around Tock.

Reply:
The comment here needs to be updated. Probably worth noting the length limitations of 4/8/16

Reply:
Updated comment. Probably this constant will be removed from the capsule and instead queried from the peripheral itself once the Networking Working Group establish a HIL for that.